example/full-example/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 0.13.0"
+
+  required_providers {
+    null = {
+      source  = "hashicorp/null"
+      version = "3.1.1"
```

Comment:
Should we allow for patches?

Reply:
Regarding the convention - I usually use `-` hyphens to resource names and `_` underscores for variables, as it makes sense to differentiate

Reply:
*as resource names I also understand local module names :)